### PR TITLE
Ankit doctest

### PR DESCRIPTION
**Pull request by ankityadav4**
## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/e61473fa10195f6ebf2308205c1e72e91b711831'>e61473fa10195f6ebf2308205c1e72e91b711831</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/public-docsite-resources/config/api-docs.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ module.exports = {
   apiJsonPaths: [
```
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29292&commitId=310fbf3ba58a3c9fabd67769f938757fb20b715b&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-TreeChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-TreeChart.Compact.default.chromium.png|91385| Changed|
|react-charting-TreeChart.Compact - RTL.default.chromium.png|91288| Changed|
|react-charting-TreeChart.Basic.default.chromium.png|91059| Changed|
|react-charting-TreeChart.Basic - Dark Mode.default.chromium.png|109565| Changed|
|react-charting-TreeChart.Basic - RTL.default.chromium.png|90986| Changed|
|react-charting-TreeChart.Compact - Dark Mode.default.chromium.png|115228| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by layershifter** [comment]
Closing for housekeeping 🏡  Feel free to re-open 👍

**Pull request closed without merging**